Correct grammar and clarify deployment sentence in documentation. (#1142)
submariner-io/submariner-website: src/content/getting-started/architecture/broker/_index.en.md

The Broker also defines a ServiceAccount and RBAC components to enable other Submariner
components to securely access the Broker's API.

> While there are no services associated with the Broker, if you use ```subctl``` to deploy the Broker, an

Reviewer: ```suggestion
While there are no services associated with the Broker, using ```subctl``` to deploy the Broker
```

> operator Pod is also deployed that installs the CRDs and the Globalnet configuration.

Reviewer: ```suggestion
also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

The Broker also defines a ServiceAccount and RBAC components to enable other Submariner
components to securely access the Broker's API.

> While there are no services associated with the Broker, using ```subctl``` to deploy the Broker
> also deploys an operator Pod that installs the CRDs and the Globalnet configuration.

Reviewer 2: ```suggestion
While there are no Services associated with the Broker, using ```subctl``` to deploy the Broker
also deploys an operator Pod that installs the CRDs and the Globalnet configuration.
```

See [the Submariner docs style guide](https://submariner.io/development/website/style-guide/) and [the section on API objects in the Kubernetes docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
